runtime/j9vm/javanextvmi.cpp

```diff
@@ -246,6 +246,7 @@ JVM_IsPreviewEnabled(void)
 	}
 	return isPreviewEnabled;
 }
+#endif /* JAVA_SPEC_VERSION >= 19 */
```

Comment:
this is not needed,   `enterVThreadTransitionCritical` and `exitVThreadTransitionCritical` is part of Java19